database/schema.sql

```diff
@@ -344,14 +344,14 @@ INSERT INTO constellation (constellation_name, constellation_short_name) VALUES
 ('Volans', 'Vol'),
 ('Vulpecula', 'Vul');
 
-INSERT INTO body (body_name, constellation_id) VALUES
-('Mercury', 86),
-('Venus', 49),
-('Mars', 38),
-('Jupiter', 78),
-('Saturn', 4),
-('Uranus', 78),
-('Neptune', 66),
-('Pluto', 4),
-('Sun', 86),
-('Moon', 72);
\ No newline at end of file
+INSERT INTO body (body_name) VALUES
+('Mercury'),
+('Venus'),
+('Mars'),
+('Jupiter'),
+('Saturn'),
+('Uranus'),
+('Neptune'),
+('Pluto'),
+('Sun'),
+('Moon');
```

Comment:
The body table no longer tracks constellation ID directly, and this change accounts for that. LGTM.